cmd/commands/server/command_server.go

```diff
@@ -76,14 +77,14 @@ func (cmd *Command) Start() (err error) {
 	providerContact, err := cmd.dialogWaiter.Start()
 
 	// if for some reason we will need truly external IP, use GetPublicIP()
-	vpnServerIP, err := cmd.ipResolver.GetOutboundIP()
+	outboundIP, err := cmd.ipResolver.GetOutboundIP()
```

Comment:
Sad that we do 2-3 requests. Is it possible to avoid it?

Reply:
Without too much refactoring? Yes - by passing outboundIP to openvpnServiceAddress function as parameter instead of looking up it again inside. I can live with this approach :)

Reply:
Fixed :) reduced ipify query to 1 request